### PR TITLE
Fix #119: Ensure a multipart Content-Type header is used when sending attachments.

### LINKED ISSUE #119: No content-type provided when sending multipart request body
Post #115 bug:

### PR DESCRIPTION
**Pull request by n1k0**
Refs #119. This is ugly, but I couldn't find any better generic way to check for a multipart request body :disappointed:  

Tested the patch on kinto-admin, solves the issue.

r=? @leplatrem

### CODE REVIEW

File: test/kinto.ini [hunk 1]
```diff
@@ -20,6 +20,8 @@ kinto.attachment.folder = {bucket_id}/{collection_id}
 kinto.attachment.keep_old_files = true
 kinto.attachment.base_path = /tmp
 
+kinto.experimental_permissions_endpoint = true
+
```
Comment (leplatrem) [inline test/kinto.ini]: nit: leftover?

File: src/http.js [hunk 2]
```diff
@@ -78,6 +78,11 @@ export default class HTTP {
     let response, status, statusText, headers, hasTimedout;
     // Ensure default request headers are always set
     options.headers = {...HTTP.DEFAULT_REQUEST_HEADERS, ...options.headers};
+    // If a multipart body is provided, remove any custom Content-Type header as
+    // the fetch() implementation will add the correct one for us.
+    if (options.body && typeof options.body.append === "function") {
+      delete options.headers["Content-Type"];
+    }
```
Comment (leplatrem) [inline src/http.js]: It is not worth than the previous `undefined` we had...

**Comment by leplatrem** [comment]
r+

File: test/kinto.ini (hunk 1 above)
Comment (n1k0) [inline test/kinto.ini]: Woops, good catch

**Pull request merged**